examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        view="item_features",
</pre><p>similarly, here <code>view</code> should be <code>item_attributes</code>?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/1022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Same as above for this change. You cannot make this change unless we change the configs.

Reply:
I think we probably should change both for the sake of clarity